classes/plugins/ThemePlugin.inc.php

```diff
@@ -571,6 +571,16 @@ public function getOptionValues($contextId) {
 						break;
 				}
 			}
+			// If the value isn't null and it's a multilingual field, then we must ensure it's an array
+			if ($optionConfig->isMultilingual && $value !== null && !is_array($value)) {
+				try {
+					$value =  json_decode((string) $value, true, 512, JSON_THROW_ON_ERROR);
+				} catch (Exception $e) {
+					// FIXME: pkp/pkp-lib#6250 Remove after 3.3.x upgrade code is removed (see also pkp/pkp-lib#5772)
```

Comment:
Just to confirm my understanding, this can safely be removed once upgrades can no longer go through the 3.3.x parts of the upgrade code?

Reply:
AFAICR, the upgrade code for 3.3 is using OJS classes to retrieve data, and that's why it's needed to keep this workaround.
From +3.4 onwards there shouldn't be `serialize()` content in the database anymore.